build/docker_config.yaml

```diff
@@ -0,0 +1,89 @@
+# Server configuration file for Archon.
```

Comment:
Is it possible to just copy the config file in `setup` and reference that? I worry a bit about needing to keep two config files in sync - I'm certain they'll accidentally diverge at some point.

Reply:
Since it's a viper underneath - it's definitely possible to merge the configs (e.g. to only have some overrides for docker) but it depends on the code and I haven't looked at this part yet. I'll try it out and will implement merging if needed.

Reply:
Done - I've added an option to provide a second config to only overwrite selected options with default viper merge function.